### PR TITLE
Added support for Windows to remove `dist` folder in unit tests.

### PR DESCRIPTION
**Pull request by royvanv**

### CODE REVIEW

File: tests/Feature/ExportTest.php [hunk 1]
```diff
@@ -17,7 +17,11 @@ protected function setUp(): void
     {
         parent::setUp();
 
-        exec('rm -r '.__DIR__.'/dist');
+        if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
```
Comment (freekmurze) [inline tests/Feature/ExportTest.php]: Make this a ternary so we can get rid of the `else` statement.

**Comment by freekmurze** [comment]
Could you fix the conflicts?

**Comment by royvanv** [comment]
> Could you fix the conflicts?

They have been fixed.

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**